packages/editor/src/dataviews/actions/export-pattern.tsx

```diff
@@ -33,15 +30,15 @@ function getJsonFromItem( item ) {
 	);
 }
 
-export const exportPatternAsJSONAction = {
+const exportPattern: Action< Pattern > = {
 	id: 'export-pattern',
 	label: __( 'Export as JSON' ),
 	supportsBulk: true,
 	isEligible: ( item ) => {
 		if ( ! item.type ) {
 			return false;
 		}
-		return item.type === PATTERN_TYPES.user;
+		return item.type === 'wp_block';
```

Comment:
We still need this because of the mapping of user patterns? When your other PR lands, it should be enough to just register it with the `wp_block` type, correct?

Reply:
Right it seems we might not need this.